Dockerfile.konflux

```diff
@@ -0,0 +1,64 @@
+ARG DEV_IMAGE=registry.redhat.io/ubi9/go-toolset@sha256:fd41c001abc243076cc28b63c409ae6d9cbcad401c8124fb67d20fe57a2aa63a
```

Comment:
if we are going to have a Containerfile specific to konflux, we can drop this change.

Reply:
YES